relay.config.js

```diff
@@ -18,5 +18,5 @@ module.exports = {
   },
   noFutureProofEnums: true,
   schema: path.join(__dirname, 'packages/server/graphql/public/schema.graphql'),
-  excludes: ['**/node_modules/**', '**/__generated__/**']
+  excludes: ['**/node_modules/**', '**/__generated__/**', '**/.*/**']
```

Comment:
While this works, I have no idea why. I thought that watchman might have a problem with newly added hidden directories, but when I add these individually, it still hangs. This pattern seems to exclude more, but I don't know what and I don't feel like debugging rust code for this right now.
The number of files generated etc. remains the same.